to_srt.py

```diff
@@ -31,7 +31,10 @@ def append_subs(start, end, prev_content, format_time):
             "end_time": convert_time(end) if format_time else end,
             "content": u"\n".join(prev_content),
         })
-
+    # displayAlign="before" means the current sub will be displayed on top, and not at bottom,
+    # so we check what's the xml:id associated to it to have an {\an8} position tag in the output file
+    check_display_align_before_re = re.compile(u'<region.*tts:displayAlign=\"before\".*xml:id=\"(.*)\"/>')
+    xml_id_display_align_before = re.search(check_display_align_before_re, text).group(1)    

```

Comment:
`.group(1)` will break the script when the `search` doesn't return anything. I'd suggest encapsulating this into a function. 

```python
def xml_id_display_align_before(text):
    """
    displayAlign="before" means the current sub will be displayed on top.
    That is and not at bottom. We check what's the xml:id associated to it
    to have an {\an8} position tag in the output file.
    """
    align_before_re = re.compile(u'<region.*tts:displayAlign=\"before\".*xml:id=\"(.*)\"/>')
    has_align_before = re.search(align_before_re, text)
    if has_align_before:
        return has_align_before.group(1)
    return u""
```
And here simply have a call like

```python
display_align_before = xml_id_display_align_before(text)
```
If you are okay with that, I can merge and apply these changes myself :)

Reply:
Looks good to me!